packages/opencensus-nodejs/test/test-plugin-loader.ts

```diff
@@ -98,6 +94,24 @@ describe('Plugin Loader', () => {
         assert.strictEqual(simpleModule.value(), 101);
       });
 
+      it('should load and patch extra plugin file', () => {
+        const pluginLoader = new PluginLoader(log, tracer);
+        assert.strictEqual(pluginLoader.plugins.length, 0);
+        pluginLoader.loadPlugins(plugins);
+        const modulename = TEST_MODULES[0];
+        const simpleModule = require(modulename);
+        assert.strictEqual(pluginLoader.plugins.length, 1);
+        assert.strictEqual(simpleModule.name(), 'patched-' + modulename);
+        assert.strictEqual(simpleModule.value(), 101);
+
+        const extraModuleName = 'extra-module';
+        const indexPath = path.dirname(require.resolve(modulename));
+        const extraFile = 'src/extra-module';
+        const extraModule = require(path.join(indexPath, extraFile));
+        assert.strictEqual(extraModule.name(), 'patched-' + extraModuleName);
+        assert.strictEqual(extraModule.value(), 121);
```

Comment:
`extraModule` is already not user-visible. I would rather see a test based around a module that uses the module exports of an internal file.

Reply:
I think if the internal file export is accessible, whats the point to load it this way?

Reply:
Test improved.